util/topgen.py

```diff
@@ -1339,39 +1339,25 @@ def render_template(template_path: str, rendered_path: Path,
             (SRCTREE_TOP / "hw/top_{}/".format(topname)).resolve()
```

Comment:
L1331-1341, no need to start a new enumerate loop. The rust file generator can sit in the same loop starting from L1291

Reply:
Yes that is true i will remove second loop.
I duplicated it only  for separation Rust from C.

Reply:
Yes that is true i will remove second loop.
I duplicated it only for separation Rust from C.

Reply:
You can move the comment at L1279 to L1292, so it is clean the first part of the loop handles C and the second part is Rust.

Reply:
Exactly